x11-plugins/wmppp/wmppp-1.3.2-r2.ebuild

```diff
@@ -8,21 +8,18 @@ MY_P=${PN}.app-${PV}
 DESCRIPTION="PPP dial control and network load monitor with NeXTStep look"
 HOMEPAGE="https://www.dockapps.net/wmpppapp"
 SRC_URI="https://www.dockapps.net/download/${MY_P}.tar.gz"
+S="${WORKDIR}/dockapps-7ec9002" # Specific to this tarball
```

Comment:
It's easier to edit the commit hash when it's the last thing in the line. This may be a styling preference - I'd suggest moving the comment on top of `S` and even including something like
```suggestion
# build dir is specific to this tarball
S="${WORKDIR}/dockapps-7ec9002"
```